src/main/resources/templates/product-form.html

```diff
@@ -0,0 +1,26 @@
+<!doctype html>
+<html lang= "en" xmlns:th="http://www.thymeleaf.org">
+<head>
+    <title>상품 페이지</title>
+</head>
+<body>
+<h1>상품 페이지</h1>
+<form th:if="${product.id == null}" th:action="@{/admin/list/add}" th:object="${product}" method="post">
+    <h3>상품 추가</h3>
+    <p>ID: <input type="number" th:field="*{id}"></input></p>
+    <p>Name: <input type="text" th:field="*{name}"></p>
+    <p>Price: <input type="number" th:field="*{price}"></p>
+    <p>Image URL: <input type="text" th:field="*{imageUrl}"></p>
+    <p><button type="submit">저장</button></p>
+</form>
+<form th:if="${product.id != null}" th:action="@{/admin/list/edit/{id}(id = ${product.id})}" th:object="${product}" method="post">
+    <h3>상품 정보 수정</h3>
+    <p>ID: <span th:text = "${product.id}"></span></p>
+    <p>Name: <input type="text" th:field="*{name}"></p>
+    <p>Price: <input type="number" th:field="*{price}"></p>
+    <p>Image URL: <input type="text" th:field="*{imageUrl}"></p>
+    <p><button type="submit">저장</button></p>
+</form>
+<a href="/admin/list">뒤로 가기</a>
+</body>
+</html>
```

Comment:
no newline at a end of file!
https://github.com/kakao-tech-campus-2nd-step2/spring-gift-product/pull/91#discussion_r1660102660